schema/utils.go

```diff
@@ -35,6 +38,23 @@ func isNumber(n interface{}) (float64, bool) {
 	}
 }
 
+// isTime takes an interface as input, and returns a time.Time if the type is
+// a string and is formatted with RFC3339
+func isTime(t interface{}) (time.Time, bool) {
+	var t2 time.Time
+	var isTime bool
+	if tStr, ok := t.(string); ok {
+		t2, err := time.Parse(time.RFC3339, tStr)
```

Comment:
Actually we do, as this method should only be called from `validateQuery`. You can thus pass it the formats defined on the `Time` validator.